packages/vapor/scss/controls/flat-select.scss

```
@@ -62,7 +64,7 @@
             }
 
             .icon {
-                fill: var(--flat-select-text-color);
+                fill: var(--color);
```

We should probably use `fill: currentColor;`
```suggestion
                fill: currentColor;
```